galera/src/replicator_smm.cpp

```diff
@@ -1513,8 +1528,7 @@ galera::ReplicatorSMM::process_conf_change(void*                    recv_ctx,
             st_.set(state_uuid_, WSREP_SEQNO_UNDEFINED);
         }
 
-        if (state_() == S_JOINING && sst_state_ != SST_NONE
-                                  && sst_state_ != SST_CANCELED)
+        if (sst_state_ == SST_WAIT && state_() == S_JOINING)
```

Comment:
Does SST_WAIT is alternative way to saying

sst_state_ != SST_NONE  && sst_state_ != SST_CANCELED